modules/geolocation/unittests.py

```diff
@@ -117,7 +117,75 @@ def test_camera_offset_from_origin_pointing_sideways_with_not_enough_downward_pi
         # Test
         np.testing.assert_equal(actual, expected)
 
+    def test_identity_mapping(self):
```

Comment:
You should put your tests in a different class as it is testing something different from mine.